Source/missiles.cpp

```diff
@@ -1560,7 +1560,10 @@ BOOL CheckIfTrig(int x, int y)
 void AddTown(int mi, int sx, int sy, int dx, int dy, int midir, char mienemy, int id, int dam)
 {
 	int i, j, k, mx, tx, ty;
+
+#ifndef HELLFIRE

```

Comment:
Seems you got more in your commit than you expecetd ;) We don't need AddTown here

Reply:
Sorry about that. Let me remove that.